app/scripts/modules/google/src/serverGroup/configure/wizard/cloneServerGroup.gce.controller.js

```diff
@@ -388,6 +388,17 @@ module.exports = angular
       gceServerGroupConfigurationService.refreshHealthChecks($scope.command);
     };
 
+    this.onEnableAutoHealingChange = function() {
+      // Prevent empty auto-healing policies from being overwritten by those of their ancestors
+      if (
+        $scope.command.stack === 'clone' &&
+        $scope.command.autoHealingPolicy != null &&
+        $scope.command.enableAutoHealing === false
+      ) {
+        $scope.command.overwriteAncestorAutoHealingPolicy = true;
```

Comment:
Could probably inline this and make `overwriteAncestorAutoHealingPolicy = <condition>`.